### ISSUE TITLE
Telegram on Linux does not play nice with Wacom tablets

### ISSUE THREAD

**Issue by wout** [telegramdesktop/tdesktop#16655]
### Steps to reproduce
1. Using a Wacom tablet
2. Click anywhere on the Telegram window

### Expected behaviour
The window should stay put and react just like if it were a regular mouse.

### Actual behaviour
 The Telegram window's top-left corner (including shadows) jumps to the cursor position.

### Configuration
**Operating system:**
elementary OS Odin Beta 2
(Built on Ubuntu 20.04.1 LTS)
(Linux 5.8.0-59-generic)

**Version of Telegram Desktop:**
2.8.9

**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:
flatpak

**Used theme**:
default

**Comment by ilya-fedin** [comment]
Please provide a way to reproduce without buying specific hardware

**Comment by wout** [comment]
@ilya-fedin I'm not sure how to respond to that, other than: shall I buy you a tablet so you can try it out for yourself?

**Comment by ilya-fedin** [comment]
I see three options: dig into the problem yourself, find a way to reproduce without buying specific hardware (some emulator or etc), wait until someone other more experienced with such hardware fix this (may never happen).

**Comment by ilya-fedin** [comment]
Honestly saying I don't really know how to debug this as well, tdesktop uses high-level Qt APIs, your issue sounds like Qt is misbehaving or something

**Comment by wout** [comment]
Ah, that makes sense. It's not the first time a Qt app misbehaves. I've got to other apps Qt apps installed, and they both have issues with HiDPI scaling. I worked around it with some env vars in the desktop files, but still, it's not ideal.

There's another issue I discovered with the Telegram app. In an attempt to reproduce the issue without a Wacom tablet (or similar), I tried enabling the touchscreen on my Dell XPS. Using the touchscreen, I can't even drag the Telegram window, while with all other apps, I can.

Then I noticed the non-standard window frame, and as it turns out, there is an option in the settings to enable the default window frame:

![image](https://user-images.githubusercontent.com/107324/126177027-3e73bf60-54e9-4ddf-8c7e-bc9fed178c68.png)

That fixed the issue.

So you're probably right. Qt isn't handling the non-standard input devices very well.
